jobs/tests.py

```diff
@@ -1,3 +1,28 @@
 from django.test import TestCase
 
-# Create your tests here.
+from model_mommy import mommy
+
+from jobs.models import Job
+
+
+class JobMethodTests(TestCase):
+
+	def setUp(self):
+		self.job = mommy.make(Job, ready_to_publish=True)
+
+	def test_is_published(self):
+		published_date_is_not_none = False
+		self.job.publish()
+		if self.job.published_date:
+			published_date_is_not_none = True
+		self.assertEqual(published_date_is_not_none, True)
+		print "Published date:{0}".format(self.job.published_date)
```

Comment:
This test can be simplified quite a lot:
```
self.assertFalse(self.job.published_date)
self.job.publish()
self.assertTrue(self.job.published_date, "Job has no published date")
```
You can use the optional last argument in the assert to print a message if the test fails.  We don't really care too much about the actual publish date (except for debugging while writing the test)